cartridges/app_onboarding/cartridge/templates/default/search/refinements/attributes/color.isml

```diff
@@ -0,0 +1,17 @@
+<li class="color-attribute ${!refinementValue.selectable ? 'disabled' : ''}">
+    <button data-href="${refinementValue.url}">
+    	<isif condition="${!require('app_onboarding/cartridge/utils/mobileChecker').isMobile()}">
```

Comment:
There is no utils folder in the standard cartridge structure. Move your util to scripts.